src/plugins/blocksmith/components/blocksmith/index.jsx

```diff
@@ -1385,9 +1388,29 @@ class Blocksmith extends React.Component {
         ...sidebarHolderStyle,
         transform: `scale(${isDocumentClick ? 0 : 1})`,
       },
-    }));
+    }), this.createPopoverToggleEvent);
+  }
+
+  createPopoverToggleEvent() {
```

Comment:
@cohencodes go ahead and put the contents of this fn inline in the above setstate callback